meteor/package-lock.json

```diff
@@ -2112,7 +2112,7 @@
 					"integrity": "sha512-kHmoybcPV8Sqy59DwNDY3Jefr64lK/by/da0ViFcuA4DH0vQg5Q6Ze5VimxkfQNSC+Mls/Kx53s7TjP1RhFEDQ=="
 				},
 				"timeline-state-resolver-types": {
-					"version": "npm:timeline-state-resolver-types@1.0.0-release37",
+					"version": "npm:@tv2media/timeline-state-resolver-types@1.0.0-release37",
```

Comment:
We are using Yarn. Why do we have a package.lock? 
It is not recommended to have both a package.lock and a yarn.lock

Reply:
I can see that on staging we have a package-lock.json and not yarn.lock under the meteor folder. But on develop we have a yarn.lock and no package-lock.json in that folder.
Some clean-up is required

Reply:
from 37.1 to develop, we have migrated from `meteor npm install` to `meteor yarn install`.